lib/base-chips/field_type.hashed-text.js

```diff
@@ -0,0 +1,44 @@
+var Sealious = require('../main.js');
```

Comment:
Should be changed to `require("sealious")`, since we're using `require-self` as of recently :)